apps/web/package.json

```diff
@@ -26,6 +26,7 @@
     "@t3-oss/env-nextjs": "^0.10.1",
     "bcrypt": "^5.1.1",
     "bullmq": "^5.8.5",
+    "classnames": "^2.5.1",
```

Comment:
No no, you have `cn` in web-ui. Don't add this dependency

Reply:
we are moving components to web, some of these will need to come with them 🤷🏼

Reply:
But I mean that `cn` is already present. You don't need this dependency

```javascript
import { cn } from '@latitude-data/web-ui'
```

Reply:
lol we are exporting cn from web-ui ok